docs/src/components/InteractivePlayground/useClampPlayground/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react';
+import { DoubleRange } from '..';
+import Example from './Example';
+
+
+const initialState = {
+  lowerBound: 100,
+  upperBound:300,
+  lowerSpringToValue:150,
+  upperSpringToValue:190
+
+};
+
+export default function useClampPlayground() {
+  const [lowerBound, setLowerBound] = useState(initialState.lowerBound);
+  const [upperBound, setUpperBound] = useState(initialState.upperBound);
+
+  const [lowerSpringToValue, setLowerSpringToValue] = useState(initialState.lowerSpringToValue);
+  const [upperSpringToValue, setUpperSpringToValue] = useState(initialState.upperBound);
+
+  const example = (
+    <Example options={{lowerBound: lowerBound, upperBound:upperBound, lowerSpringToValue, upperSpringToValue}}/>
+  )
+  
+  const controls = (
+  <>
+    <DoubleRange
+      label="Clamp limits"
+      min={0}
+      max={400}
+      step={10}
+      value={[lowerBound, upperBound]}
+      onChange={[setLowerBound, setUpperBound]}
+    />
+    <DoubleRange
+      label="Scope of movement"
```

Comment:
I don't like this label as the scope of movement says nothing. This is the value to which the spring is finishing at 🤔